crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -305,6 +311,26 @@ impl PgIndexerStore {
             )
     }
 
+    async fn get_min_cp_and_tx_for_epoch(&self, epoch: u64) -> Result<(u64, u64), IndexerError> {
+        use diesel_async::RunQueryDsl;
+        let epoch = epoch.saturating_sub(1) as i64;
```

Comment:
to get min cp and tx for epoch N, we will need to read checkpoints of epoch N - 1, and when we do this, checkpoints of epoch N - 1 might have been pruned?